db/migrate/20171030200258_add_office.rb

```diff
@@ -0,0 +1,7 @@
+class AddOffice < ActiveRecord::Migration[5.1]
```

Comment:
Just an FYI: Migrations are conventionally named starting with `Create` when you create a new table and `Add` when you are adding to an existing table. This would be `CreateOffices` (pluralized). If you use this naming convention then you can also generate the entire migration file from the command line: `rails g migration CreateOffices name:string`.

A decent overview of what's possible from the command line: https://www.ralfebert.de/snippets/ruby-rails/models-tables-migrations-cheat-sheet/